README.md

```diff
@@ -2,6 +2,20 @@
 
 The DHL API Javascript client implementation.
 
+```javascript
+const dhlAPI = require("dhl-api");
+
+// instance the DHL client with defaults
+const dhl = new dhlApi.API({
+    username: "mydhlaccount",
+    password: "mydhlaccountpassword",
+    license: "3F9955C3F789C255"
+});
```

Comment:
These parameters are wrong. Take a look at the mandatory ones (the ones with no default for [UPS  here](https://github.com/ripe-tech/ups-api-js/blob/master/js/base.js#L31-L33))

Now check the [constructor for the DHL one](https://github.com/ripe-tech/dhl-api-js/blob/master/js/base.js#L10). Notice we can give username password or the token itself (which defaults to user + password).

I would therefore document username and password only.